src/System.Management.Automation/engine/LanguagePrimitives.cs

```diff
@@ -1,6 +1,8 @@
 // Copyright (c) Microsoft Corporation.
```

Comment:
@powercode, your last commit had 6 failures in `PowerShell-CI-windows`
(These are 5 of the failures)

TabCompletion.CIM cmdlet completion tests.CIM cmdlet input 'Get-CimInstance -Namespace root/Inter' should successfully complete
```powershell
Expected strings to be the same, but they were different.
String lengths are both 12.
Strings differ at index 5.
Expected: 'root/Interop'
But was:  'root/interop'
at <ScriptBlock>, D:\a\1\s\test\powershell\Host\TabCompletion\TabCompletion.Tests.ps1: line 1252
1252:             $res.CompletionMatches[0].CompletionText | Should -BeExactly $expected
```
TabCompletion.CIM cmdlet completion tests.CIM cmdlet input 'Get-CimInstance -Namespace root/Int*ro' should successfully complete
```powershell
Expected strings to be the same, but they were different.
String lengths are both 12.
Strings differ at index 5.
Expected: 'root/Interop'
But was:  'root/interop'
at <ScriptBlock>, D:\a\1\s\test\powershell\Host\TabCompletion\TabCompletion.Tests.ps1: line 1252
1252:             $res.CompletionMatches[0].CompletionText | Should -BeExactly $expected
```
TabCompletion.CIM cmdlet completion tests.CIM cmdlet input 'New-CimInstance -Namespace root/Inter' should successfully complete
```powershell
Expected strings to be the same, but they were different.
String lengths are both 12.
Strings differ at index 5.
Expected: 'root/Interop'
But was:  'root/interop'
at <ScriptBlock>, D:\a\1\s\test\powershell\Host\TabCompletion\TabCompletion.Tests.ps1: line 1252
1252:             $res.CompletionMatches[0].CompletionText | Should -BeExactly $expected
```
TabCompletion.CIM cmdlet completion tests.CIM cmdlet input 'Invoke-CimMethod -Namespace root/Inter' should successfully complete
```powershell
Expected strings to be the same, but they were different.
String lengths are both 12.
Strings differ at index 5.
Expected: 'root/Interop'
But was:  'root/interop'
at <ScriptBlock>, D:\a\1\s\test\powershell\Host\TabCompletion\TabCompletion.Tests.ps1: line 1252
1252:             $res.CompletionMatches[0].CompletionText | Should -BeExactly $expected
```
TabCompletion.CIM cmdlet completion tests.CIM cmdlet input 'Get-CimClass -Namespace root/Inter' should successfully complete
```powershell
Expected strings to be the same, but they were different.
String lengths are both 12.
Strings differ at index 5.
Expected: 'root/Interop'
But was:  'root/interop'
at <ScriptBlock>, D:\a\1\s\test\powershell\Host\TabCompletion\TabCompletion.Tests.ps1: line 1252
1252:             $res.CompletionMatches[0].CompletionText | Should -BeExactly $expected
```